public/gamecompare.php

```diff
@@ -58,6 +58,13 @@
     }
 }
 
+sanitize_outputs(
+    $gameTitle,
+    $consoleName,
+    $gameIcon,
+    $user,
+);
```

Comment:
I'm leary of this change. These values are passed to `GetGameAndTooltipDiv` and `GetUserAndTooltipDiv`, which will likely sanitize them again.

Reply:
Good point - looks good so far though? 

![Screenshot 2022-06-03 at 20 26 08](https://user-images.githubusercontent.com/1280590/171924258-041ed1c8-d854-4422-b82b-b89fd7819851.png)
![Screenshot 2022-06-03 at 20 26 14](https://user-images.githubusercontent.com/1280590/171924272-fc4e4383-19e0-4bb3-84de-dded4932afd9.png)